snuba/datasets/storages/discover.py

```diff
@@ -96,6 +97,7 @@
                 "project_id",
             ]
         ),
+        MandatoryConditionEnforcer({"project_id", "timestamp"}),
```

Comment:
I know why you put this here, but I wanted to ask about this: these columns are already represented in the Entity, as `required_filter_columns` / `required_time_column`. Is there some way we can have these columns defined in one place, instead of hardcoded in the entity and the storage?

Reply:
I don't think we can. 
The required filter columns and required time columns in the entities are about the logical model (entities) while this is on a storage. While it is improbable that we would translate the project_id column in something different, there is no such guarantee.